docs/windows.md

```diff
@@ -125,7 +125,10 @@ document.
 kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea.yml
 ```
 
-#### Add Windows kube-proxy DaemonSet
+#### Add Windows kube-proxy DaemonSet (for Kubernetes versions prior to 1.26)
+
+Windows kube-proxy will no longer be supported by Antrea starting from Kubernetes 1.26. This decision is based on the deprecation of the userspace mode,
+which Antrea heavily relies on. You should skip this step on the latest kubernetes cluster.
```

Comment:
Add a sentence to explain that Antrea ProxyAll is used as a replacement of kube-proxy ?

Reply:
Updated.